documentation/whatsnew/webservices/webservices-news-1.8.0.Beta1.adoc

```diff
@@ -0,0 +1,38 @@
+= WebServices Tools 1.8.0.Beta1 What's New
+:page-layout: whatsnew
+:page-component_id: webservices
+:page-component_version: 1.8.0.Beta1
+:page-product_id: jbt_core 
+:page-product_version: 4.3.0.Beta1
+
+== Content assist for methods in Websocket endpoint classes
+
+Content assist for methods in Websocket endpoint classes is added.
+
+Websocket endpoint classes are classes annotated with javax.websocket.server.ServerEndpoint or javax.websocket.ClientEndpoint.
+They can have methods annotated with *@OnClose*, *@OnError*, *@OnMessage*, *@OnOpen*.
+
+Content assist provides options:
```

Comment:
could you add one example of what code it will actually generate  ? 

since without that this description makes it sound like it is just doing normal content assist.